### PR TITLE
Dev 77 menzorg

### PR DESCRIPTION
**Pull request by Menzorg**

### CODE REVIEW

File: electron/electron-builder.config.json [hunk 1]
```diff
@@ -9,25 +9,47 @@
     "capacitor.config.*",
     "app/**/*"
   ],
-  "publish": {
-    "provider": "github"
-  },
+  "publish": null,
+  "afterPack": "./copy-deep.js",
```
Comment (ivansglazunov) [inline electron/electron-builder.config.json]: Please rename to `recovery-after-electron-build.js`

File: electron/package.json [hunk 2]
```diff
@@ -13,10 +14,13 @@
   "license": "MIT",
   "main": "build/src/index.js",
   "scripts": {
-    "build": "tsc && electron-rebuild",
+    "build": "tsc",
```
Comment (ivansglazunov) [inline electron/package.json:17]: why deleted?

**Review by ivansglazunov — COMMENTED** [review]
public/logo.png must not be transparent

File: electron/package.json (hunk 2 above)
Comment (Menzorg) [reply electron/package.json:17]: electron build says that build is enoth, so i will delete electron-rebuild form deps too

File: electron/electron-builder.config.json (hunk 1 above)
Comment (Menzorg) [reply electron/electron-builder.config.json]: done

**Pull request merged**